pkg/apis/sources/v1alpha1/cron_job_types.go

```diff
@@ -55,13 +56,15 @@ var (
 const (
 	// CronJobEventType is the CronJob CloudEvent type.
 	CronJobEventType = "dev.knative.cronjob.event"
-	// CronJobEventSource is the CronJob CloudEvent source.
-	CronJobEventSource = "/CronJob"
 )
 
+// CronJobEventSource returns the CronJob CloudEvent source.
+func CronJobEventSource(namespace, cronJobName string) string {
+	return fmt.Sprintf("/apis/v1/namespaces/%s/cronjobsources/%s", namespace, cronJobName)
```

Comment:
:thinking:  I guess... this is mostly making sense in the context of "where" the "job" was trigger.

Wondering... does it need to include `sources`, in `cronjobsources` ? (I guess a lot will also name it `my-cronjobsource` ) :-)

Reply:
The naming was a suggestion from @n3wscott. I'm fine with either. I didn't want to have the previous /CronJob though, as it doesn't add much (you can just get that with the type).

Reply:
> Wondering... does it need to include sources, in cronjobsources ? (I guess a lot will also name it my-cronjobsource ) :-)

It needs `cronjobsources` because that is the resource name.